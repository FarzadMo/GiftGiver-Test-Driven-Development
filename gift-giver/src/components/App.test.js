import React from "react";
import { shallow } from "enzyme";
import App from "./App";

const app = shallow(<App />);

//Unit test

it("rendors correctly", () => {
  expect(app).toMatchSnapshot();
});
