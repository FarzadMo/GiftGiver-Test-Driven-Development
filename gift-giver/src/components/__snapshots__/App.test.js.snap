// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App rendors correctly 1`] = `ShallowWrapper {}`;

exports[`rendors correctly 1`] = `ShallowWrapper {}`;
